### DISCUSSION TITLE
Add the very OG osu! old skin back in 2007

### DISCUSSION THREAD

**Discussion by tematikkp** [ppy/osu#20659] [Ideas] [answered]
Hello! I'm suggesting that there should be an osu! old skin, to bring old osu! players' throwback to legacy skin.

**Comment by peppy** [comment, marked as answer]
For sure, this is planned.

**Reply by SirHonkkk** [reply]
heres an idea: you could call it osu! elite, to symbolize the inspiration from elite beat agents

**Reply by SSolarite** [reply]
its been 3 years bro can you add the skin already

**Reply by Joehuu** [reply]
Please note that other feature requests have way more upvotes. And the upvotes here are probably driven by nostalgia.

There are also skins out there that do this: https://osu.ppy.sh/community/forums/topics/380575 (not official)

Maybe officially, it can be done as a 20th anniversary thing, dunno.

**Reply by tadatomix** [reply]
Strange idea, but it would be interesting if it will also add no title screen at all. So it'll just show you a menu with a random song from your listing

**Reply by SSolarite** [reply]
> Please note that other feature requests have way more upvotes. And the upvotes here are probably driven by nostalgia.
> 
> There are also skins out there that do this: https://osu.ppy.sh/community/forums/topics/380575 (not official)
> 
> Maybe officially, it can be done as a 20th anniversary thing, dunno.

True.........

**Reply by SSolarite** [reply]
> Strange idea, but it would be interesting if it will also add no title screen at all. So it'll just show you a menu with a random song from your listing

... there was a title screen back in 2007 though

**Reply by tadatomix** [reply]
My bad, I meant no intro theme and animation. It was just showing a title screen with a random music in your library

**Comment by ziv-vy** [comment]
The taiko skin is essentially just a ripoff of the Tatsujin look, that would be copyright hell. 

I think that the classic and prehistoric CTB skin are both identical, so will only benefit mania and probably standard.

I think the copyright issue applies to std too, but the Ouendan devs would probably accept this as they stated that they do appreciate osu!.

Would love to have such a skin.

**Reply by peppy** [reply]
No copyright assets were used in taiko or osu! skins.

**Reply by SSolarite** [reply]
hi peppy, where did you find the hitsound effects? on https://strudel.cc the openhat sound sounds the same as drum hitwhistle

**Comment by cl8n** [comment]
I want to add this but first I'm trying to get more (gameplay) legacy skin features working and do some restoration of very old visual features that have been dropped over the years. the old skin doesn't really have the same vibe to me without all of these

my goals before adding the skin:
- slider points (#34606)
- geki/katu
- combo fire (#34529)
- missing slider styles (1 and 3, 1 would be a good default for v1 skins)
- old hit animations skin option (again, maybe default to old style for v1?)

other things on my mind but not sure if I'll get to it:
- countdown (#21009)
- combo bursts (#17616)
- pause menu
- skip button
- section pass/fail
- taiko flowers

i'll be slowly working thru these regardless of if they end up in the official project cuz it's been fun so far, but expect some PRs soon :^)

**Reply by tadatomix** [reply]
I think slider styles may be replicated if slider texturing become a thing as seen here 
https://github.com/ppy/osu/discussions/27795#discussioncomment-13893009

**Reply by cl8n** [reply]
that method wont work for sliderstyle 1 due to how it handles corners & self-overlap unfortunately. the segments need to be flexible in ways a static texture couldn't support either. super cool though

i have sliderstyle 1 working already, but the code sucks and im having a hard time refactoring to fit it in well. if any devs reading this, it's because my new path doesnt inherit from the framework Path, and the whole heirarchy of Slider classes has Path very baked in. I also need to handle snaking internally whereas currently sliders generate a new set of vertices for the snaked portion

**Reply by tadatomix** [reply]
May I ask for the list of slider styles then, since I don't really understand what do you mean, please?

**Reply by peppy** [reply]
combo bursts have an existing PR that could be of some use https://github.com/ppy/osu/pull/17616

**Reply by WebKun** [reply]
What about fruit rain in CTB

**Comment by Number-3434** [comment]
how is it actually worse than Classic skin

EDIT: guys im joking ok

**Reply by Number-3434** [reply]
yes

**Reply by SSolarite** [reply]
are you stupid

**Reply by Number-3434** [reply]
probably xD

**Reply by ziv-vy** [reply]
bruh
